packages/office-ui-fabric-react/src/components/Icon/examples/Icon.SvgFactory.Example.tsx

```diff
@@ -1,24 +1,74 @@
 import * as React from 'react';
-import { OneDriveIcon, YammerIcon, BorderBlindsIcon } from '@fluentui/react-icons';
-import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import * as ReactIcons from '@fluentui/react-icons';
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
 
-const rootClass = mergeStyles({
-  display: 'flex',
-  selectors: {
-    '> *': {
-      height: 50,
-      width: 50,
-      marginRight: 25,
+const classes = mergeStyleSets({
+  root: {
+    selectors: {
+      '> *': {
+        height: 50,
+        width: 50,
+        marginRight: 25,
+      },
     },
   },
+  cell: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    margin: '80px',
+    float: 'left',
+  },
+  code: {
+    background: '#f2f2f2',
+    borderRadius: '4px',
+    padding: '4px',
+  },
+  navigationText: {
+    width: 100,
+    margin: '0 5px',
+  },
 });
 
+const icons = Object.keys(ReactIcons).reduce((acc: React.FC[], exportName) => {
```

Comment:
We may want to not load all 2000+ icons here but rather a few key ones, or have a separate page as an icon viewing page.

Reply:
Note that this page ends up showing in both the public doc site and the snapshot tests.

Reply:
Actually I am showing 100 icons at a time, as I have pagination. I didn’t know what’s the best way of testing all icons with screenshots, so let’s sync ok that.